src/compiler/checker.ts

```diff
@@ -22946,15 +23202,11 @@ namespace ts {
                     clauseWitnesses = <string[]>switchWitnesses.slice(clauseStart, clauseEnd);
                     switchFacts = getFactsFromTypeofSwitch(clauseStart, clauseEnd, <string[]>switchWitnesses, hasDefaultClause);
                 }
-                if (hasDefaultClause) {
-                    return filterType(type, t => (getTypeFacts(t) & switchFacts) === switchFacts);
-                }
+
                 /*
-                  The implied type is the raw type suggested by a
-                  value being caught in this clause.
+                  The implied type is the raw type suggested by a value being caught in this clause.
 
-                  When the clause contains a default case we ignore
-                  the implied type and try to narrow using any facts
+                  When the clause contains a default case we ignore the implied type and try to narrow using any facts

```

Comment:
Can you revert all the unnecessary/unrelated changes? I can’t stress enough how much I’m trying to make this PR _smaller_ 😅